packages/riipen-ui/src/components/Drawer.test.jsx

```diff
@@ -0,0 +1,177 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import Drawer from "./Drawer.jsx";
+
+describe("<Drawer>", () => {
+  it("renders without errors", () => {
+    let error;
+    try {
+      mount(<Drawer />);
+    } catch (e) {
+      error = e;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("snapshot test when open is true and onClose is provided", () => {
+    const open = true;
+    const onClose = jest.fn();
```

Comment:
I just realized `onClose` is required for `<ClickAway>` but not in `Drawer`. I think we have 2 options here:

1. Make `onClose` a required prop for `<Drawer>`
2. Only render the `<ClickAway>` if `onClose` is provided, else just render the children.

Both have pros and cons. I think number 1 makes more sense to me.